elyra/pipeline/component_parser_airflow.py

```diff
@@ -37,6 +37,8 @@ def parse(self, registry_entry) -> List[Component]:
         components: List[Component] = list()
```

Comment:
Please update the return type hint to `-> Optional[List[Component]]:` since `None` can now be returned.